### PR TITLE
Feature/uu 625 (datagrid) Email

### PR DESCRIPTION
**Pull request by whispyy**
oui-datagrid :
- email task
- email redirection
- email filter
- email-delegate filter

### CODE REVIEW

File: client/app/email-domain/delegate/filter/email-domain-delegate-filter.controller.js [hunk 1]
```diff
@@ -17,32 +17,22 @@ angular.module("App").controller(
             this.accounts = this.$scope.ctrlEmailDelegate.emails || [];
             this.currentAccount = this.$scope.ctrlEmailDelegate.currentViewData || null;
             this.filtersDetails = [];
```
Comment (JDownloader) [inline client/app/email-domain/delegate/filter/email-domain-delegate-filter.controller.js]: ```filtersDetails ``` is probably unused

File: client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js [hunk 2]
```diff
@@ -20,7 +20,6 @@ angular.module("App").controller(
         $onInit () {
             this.loading = {
```
Comment (JDownloader) [inline client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js]: ```exportCSV``` and ```redirections``` are assigne multiple time but never used.

File: client/app/email-domain/task/email-domain-task.html [hunk 3]
```diff
@@ -1,73 +1,30 @@
...
+                  data-ng-if="$row.status"
+                  data-ng-class="{
+                    'label-info': $row.status !== 'ERROR' && $row.status !== 'CANCELLED' && $row.status !== 'DONE',
+                    'label-danger': $row.status === 'ERROR',
+                    'label-default': $row.status === 'CANCELLED',
+                    'label-success': $row.status === 'DONE' }"
+                  data-ng-bind="tr('email_tab_TASK_status_' + $row.status)"></span>
+            <span class="label label-info"
+                  data-ng-if="!$row.status"
+                  data-ng-bind="tr('email_tab_TASK_status_DOING')"></span>
```
Comment (JDownloader) [inline client/app/email-domain/task/email-domain-task.html]: data-i18n-static would be a better option and single binded

File: client/app/email-domain/task/email-domain-task.html [hunk 4]
```diff
@@ -1,73 +1,30 @@
...
+        </oui-column>
+        <oui-column data-title="tr('email_tab_TASK_table_state')">
+            <span class="label"
+                  data-ng-if="$row.status"
+                  data-ng-class="{
+                    'label-info': $row.status !== 'ERROR' && $row.status !== 'CANCELLED' && $row.status !== 'DONE',
+                    'label-danger': $row.status === 'ERROR',
+                    'label-default': $row.status === 'CANCELLED',
+                    'label-success': $row.status === 'DONE' }"
+                  data-ng-bind="tr('email_tab_TASK_status_' + $row.status)"></span>
```
Comment (JDownloader) [inline client/app/email-domain/task/email-domain-task.html]: data-i18n-static would be a better option and single binded

File: client/app/email-domain/delegate/filter/email-domain-delegate-filter.html [hunk 5]
```diff
@@ -13,75 +13,24 @@ <h2>
...
-            </div>
-        </div>
+        <oui-datagrid data-rows="ctrlEmailDelegateFilter.filters"
+                      data-row-loader="ctrlEmailDelegateFilter.transformItem($row)">
+            <oui-column data-title="tr('email_tab_table_header_filter_name')" data-property="name" data-sortable="asc"></oui-column>
+            <oui-column data-title="tr('email_tab_table_header_filter_priority')" data-property="priority" data-sortable></oui-column>
+            <oui-column data-title="tr('emails_common_status')">
+                <span class="label"
+                      data-ng-class="{'label-success': $row.active, 'label-danger': !$row.active}"
+                      data-ng-bind="tr('email_tab_filters_status_' + $row.active)"></span>
```
Comment (JDownloader) [inline client/app/email-domain/delegate/filter/email-domain-delegate-filter.html]: data-i18n-static would be a better option and single binded

File: client/app/email-domain/email/filter/email-domain-email-filter.html [hunk 6]
```diff
@@ -12,71 +12,31 @@ <h2>
...
-            </div>
-        </div>
+        <oui-datagrid data-rows="ctrlEmailDomainEmailFilter.filters"
+                      data-row-loader="ctrlEmailDomainEmailFilter.transformItem($row)">
+            <oui-column data-title="tr('email_tab_table_header_filter_name')" data-property="name" data-sortable="asc"></oui-column>
+            <oui-column data-title="tr('email_tab_table_header_filter_priority')" data-property="priority" data-sortable></oui-column>
+            <oui-column data-title="tr('emails_common_status')">
+                <span class="label"
+                      data-ng-class="{'label-success': $row.active, 'label-danger': !$row.active}"
+                      data-ng-bind="tr('email_tab_filters_status_' + $row.active)"></span>
```
Comment (JDownloader) [inline client/app/email-domain/email/filter/email-domain-email-filter.html]: data-i18n-static would be a better option and single binded

File: client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js (hunk 2 above)
Comment (whispyy) [reply client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js]: saw with u. used in view to disable buttons.

File: client/app/email-domain/task/email-domain-task.html (hunk 4 above)
Comment (whispyy) [reply client/app/email-domain/task/email-domain-task.html]: saw with u too. Canno't use this due to $row

File: client/app/email-domain/delegate/filter/email-domain-delegate-filter.html (hunk 5 above)
Comment (whispyy) [reply client/app/email-domain/delegate/filter/email-domain-delegate-filter.html]: same

File: client/app/email-domain/email/filter/email-domain-email-filter.html (hunk 6 above)
Comment (whispyy) [reply client/app/email-domain/email/filter/email-domain-email-filter.html]: same

File: package.json [hunk 7]
```diff
@@ -76,7 +76,7 @@
     "less-plugin-remcalc": "0.0.1",
     "ovh-protractor-jasmine2-logs-reporter": "0.0.1",
     "ovh-ui-kit": "^2.0.0",
-    "ovh-ui-kit-bs": "git+https://github.com/ovh-ux/ovh-ui-kit-bs#v1.0.1",
+    "ovh-ui-kit-bs": "git+https://github.com/ovh-ux/ovh-ui-kit-bs#v1.0.3",
```
Comment (AxelPeter) [inline package.json]: Could be updated to `"ovh-ui-kit-bs": "^1.0.3"` :)

**Pull request merged**